### PR TITLE
[DATA] Data recipe for creating dataset from MongoDB

### PR DESCRIPTION
**Pull request by nkpng2k**
Adds data recipe for creating dataset from MongoDB

### CODE REVIEW

**Comment by surenH2oai** [comment]
@JHerna17 can this be closed?  anything else is needed other than verification?

**Comment by nkpng2k** [comment]
@JHerna17 I see your updates to the code. but I think it might be safer to have an complete readme. Then we can merge

cc @surenH2oai

File: data/databases/create_dataset_from_mongodb_collection.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Author: Nicholas Png
+# Created: 31/01/2020
+# Last Updated: 20/02/2020
+
+import datatable as dt
+import pandas as pd
+from h2oaicore.data import CustomData
+
+
+_global_modules_needed_by_name = ["pymongo", "dnspython"]
```
Comment (mmalohlava) [inline data/databases/create_dataset_from_mongodb_collection.py]: Why do you need `_global_modules_needed_by_name` and then also `_modules_needed_by_name`?

Comment (nkpng2k) [reply data/databases/create_dataset_from_mongodb_collection.py]: true... I think I only need the first. But `pymongo` is not installed without `_global_modules_needed_by_name`

**Pull request merged**